test/basic.test.js

```diff
@@ -349,6 +349,20 @@ test('basic prettifier tests', (t) => {
     t.end()
   })
 
+  t.test('handles numbers', (t) => {
+    const pretty = prettyFactory()
+    let formatted = pretty(2)
+    t.is(formatted, '2\n')
+
+    formatted = pretty(-2)
+    t.is(formatted, '-2\n')
+
+    formatted = pretty(0.2)
+    t.is(formatted, '0.2\n')
+
```

Comment:
can you add a test for `Infinity` and `NaN` (since typeof NaN is number)

Reply:
I predict these will fail

Reply:
here you go

Reply:
I predict they will not parse as JSON and thus will not reach this check at all https://tools.ietf.org/html/rfc7159#section-3

Reply:
you predict right 😆